draft-fett-oauth-sd-jwt-vc-types.md

```diff
@@ -402,25 +406,91 @@ MUST contain at least one of the following properties:
 
 # Claim Metadata {#ClaimMetadata}
 
-The `claims` property is an object containing information about particular claims
-for displaying and validating the claims.
+The `claims` property is an array of objects containing information about
+particular claims for displaying and validating the claims.
 
-The object MAY contain a property for each claim that is supported by the type.
-The property name MUST be a JSON Pointer [@!RFC6901] pointing to the claim in the
-credential. The property value MUST be an object containing the following
-properties:
+The object MAY contain an object for each claim that is supported by the type.
+The property value MUST be an object containing the following properties:
 
+- `path`: An array indicating the claim or claims that are being addressed, as
+  described below. This property is REQUIRED.
 - `display`: An object containing display information for the claim, as
   described in (#ClaimDisplayMetadata). This property is OPTIONAL.
 - `verification`: A string indicating how the claim is verified, as described in
   (#ClaimVerificationMetadata). This property is OPTIONAL.
 - `sd`: A string indicating whether the claim is selectively disclosable, as
   described in (#ClaimSelectiveDisclosureMetadata). This property is OPTIONAL.
 
-Note: The JSON Pointer MUST point to the respective claim as if all selectively
-disclosable claims were disclosed to the Verifier. That means that a consuming
-application which does not have access to all disclosures may not be able to
-validate the claim.
+## Claim Path {#ClaimPath}
+
+The `path` property MUST be a non-empty array of strings, `null` values, or
+non-negative integers. It is used to select a particular claim in the credential
+or a set of claims. A string indicates that the respective key is to be
+selected, a `null` value indicates that all elements of the currently selected
+array(s) are to be selected, and a non-negative integer indicates that the
+respective index in an array is to be selected.
+
+The following shows a non-normative, simplified example of a credential:
+
+```json
+{
+  "name": "Arthur Dent",
+  "address": {
+    "street_address": "42 Market Street",
+    "city": "Milliways",
+    "postal_code": "12345"
+  },
+  "degrees": [
+    {
+      "type": "Bachelor of Science",
+      "university": "University of Betelgeuse"
+    },
+    {
+      "type": "Master of Science",
+      "university": "University of Betelgeuse"
+    }
+  ],
+  "nationalities": ["British", "Betelgeusian"]
+}
+```
+
+The following shows examples of `path` values and the respective selected
+claims:
+
+- `["name"]`: The claim `name` with the value `Arthur Dent` is selected.
+- `["address", "street_address"]`: The claim `street_address` with the value
+  `42 Market Street` is selected.
+- `["degrees", null, "type"]`: All `type` claims in the `degrees` array are
+  selected.
```

Comment:
I wonder if it might be worthwhile to add one with just `["degrees"]` or `["address"]` to give an example of selecting and providing metadata about the higher level claim of a nested claim structure?

Reply:
Something like
- `["address"]`: The claim `address` with its sub-claims as the value is selected.

Reply:
Done!